### DISCUSSION TITLE
Motif Analysis for a small peak set

### DISCUSSION THREAD

**Discussion by Chhiring-Lama** [stuart-lab/signac#905] [Q&A] [answered]
Hi, I am looking at gene-peak linkage for specific genes, and want to see what motifs are enriched in the significantly correlated peaks. But, the problem is that I am testing few (say 3 or 4) peaks to thousands of background peaks. Can I trust the p-value and fold enrichment in this case. Is there any other approach I should take. What do you suggest?

**Comment by timoast** [comment, marked as answer]
I would not recommend running motif enrichment with such a small number of peaks, with such a small sample size the fold enrichment is not reliable

**Reply by Chhiring-Lama** [reply]
Can I use only the "observed" column that suggests number of peak in the peak set with certain motif to see if the motif is enriched in the region? For instance, categorizing any motifs present in at least 1 peak as being enriched in the region.

**Reply by timoast** [reply]
With a small sample size it's difficult to say anything about the significance of any enrichment

**Comment by waltno** [comment]
I am also trying to do a similar thing. Is it reasonable to infer motif enrichment by subsetting the motif matrix by the linked peaks of interest? Then motifs with higher counts might be potential regulators?
